package.json

```diff
@@ -50,11 +51,12 @@
     "emojis": "^1.0.10",
     "eslint-plugin-github": "^4.3.5",
     "eslint-plugin-security": "^1.4.0",
+    "kafkajs": "^2.2.0",
```

Comment:
done, I have reverted the changes ig these changes happen automatically when I install the `better-ajv-errors` module because I was getting errors on doing npm run dev in these modules, no worries I reverted them back to the original versions